queries/faust/highlights.scm

```diff
@@ -0,0 +1,317 @@
+; See: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#highlights
```

Comment:
I think it's better to remove these comments here in our repo (which already includes the guide -- which may change over time!)

Reply:
Good call, removed these